pkg/cluster/controllers/database/readiness.go

```diff
@@ -161,8 +165,34 @@ func (p *PostgreSQLController) GetInClusterDatabaseConn(ctx context.Context, har
 	return conn, nil
 }
 
+type SecretsTemplate string
+
+func (f *SecretsTemplate) Format(a ...string) string {
+	res := string(*f)
+
+	for i := 0; i < len(a); i += 2 {
+		res = strings.Replace(res, "{"+a[i]+"}", a[i+1], -1)
+	}
+
+	return res
+}
```

Comment:
@steven-zou this is a blunt copy from https://github.com/zalando/postgres-operator/blob/0745ce7cce63ab6431103d27eb08e54ec47d2b15/pkg/util/config/util.go#L232

does this need to be re-written to avoid licensing issues?

Reply:
@Ghostbaby maybe you could say sth about this?